### DISCUSSION TITLE
Pregunta sobre los equipos FrontEnd y BackEnd

### DISCUSSION THREAD

**Discussion by JulioRamnses** [Launch-X-Latam/MisionFrontEnd#196] [Q&A]
Hola tengo una pregunta sobre los equipos de frontend y backend y es la siguiente

y es que no me aparece que estoy en el grupo de frontend ni el de backend( en el de backend puede que sea por las solicitudes ), en teoría debería de aparecer esto 
![image](https://user-images.githubusercontent.com/114268025/195707938-6105407a-2119-431c-aa8d-b57b2ffa8754.png)
pero no me aparece nada, solo esto 
![image](https://user-images.githubusercontent.com/114268025/195708031-b90adb20-198b-4f72-b9e0-15c106df2ddd.png)
 y no se si fue por que me salte un paso o hice algo mal, a alguien mas le pasa? o solo a mi?

disculpen las molestias y que tengan buena tarde, día o noche

**Comment by Cerecero** [comment]
Es todo lo que aparece? Has intentado dar scroll hasta abajo? Los teams estan hasta el final de la pagina directamente debajo del
 `Learn more.`

**Reply by JulioRamnses** [reply]
si es todo lo que aparece, y cuando le doy a _Learn more._ me manda al siguiente enlace 
https://docs.github.com/es/organizations/organizing-members-into-teams

**Reply by Cerecero** [reply]
Creo que es momento de que contactes con soporte de innovaccion.

**Reply by rpajares** [reply]
y que haces en un grupo u otro?, no es lo mismo para todos?

**Comment by doguedogue** [comment]
Te falta inscribirte, ahi está la forma, vamos tu puedes 🚀🚀🚀

https://launchxauth.azurewebsites.net/jointeams

**Reply by JulioRamnses** [reply]
ya hice eso, pero me sigue marcando lo mismo :c

**Reply by WhoPotz** [reply]
Creo que a la mayoria no le aparece, talvez es por que solo los administradores lo pueden ver y no lo han publicado aun.
